compiler/compiler-settings.cpp

```diff
@@ -243,6 +243,12 @@ void CompilerSettings::init() {
   }
   functions_file.value_ = get_full_path(functions_file.get());
 
+  if (tracked_builtins_file.value_.empty()) {
+    if (!is_k2_mode) {
+      tracked_builtins_file.value_ = kphp_src_path.get() + "builtin-functions/tracked/k2-unsupported.txt";
+    }
```

Comment:
Seems strange a little bit. Really, it will always use `k2-unsupported.txt` file from the repo, this is not flexible

Reply:
make it as env and option